python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -56,7 +56,8 @@ async def complete_chat(
         settings.messages = self._prepare_chat_history_for_request(chat_history)
         settings.stream = False
         async with AsyncSession(self.session) as session:
-            async with session.post(str(self.url), json=settings.prepare_settings_dict()) as response:
+            request = settings.prepare_settings_dict(model=self.ai_model_id)
```

Comment:
this can also be reverted once the settings model is set.

Reply:
I changed it back and followed the guidance in other comments.